package.json

```diff
@@ -14,6 +14,11 @@
   "author": "Open Terms Archive",
   "scripts": {
     "build": "hugo --minify",
-    "start:dev": "hugo serve --watch --logLevel debug --disableFastRender"
+    "test": "npm run lint:markdown",
```

Comment:
At the moment, this script is of no real value to me, but I assume you'll improve it by adding more tests later.

Reply:
I always find it useful to set up `test` as an npm script since it's a [lifecycle script](https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts) 🙂